web-ng/src/app/services/router/router.service.ts

```diff
@@ -162,13 +145,14 @@ export class NavigationService {
   /**
    * Navigate to the current URL, updating the feature id in the URL fragment.
    */
-  setFeatureId(id: string | null) {
-    const updates: { [key: string]: string | null } = {};
-    updates[NavigationService.FEATURE_ID_FRAGMENT_PARAM] = id;
-    if (this.getObservationId()) {
-      updates[NavigationService.OBSERVATION_ID_FRAGMENT_PARAM] = null;
-    }
-    this.setFragmentParam(Map(updates));
+  setFeatureId(id: string) {
```

Comment:
I think this would be `selectFeature(id: string)` given the new semantics of this class.